### ISSUE TITLE
Couldn't find ffmpeg or avconv

### ISSUE THREAD

**Issue by girishgargcool** [3b1b/manim#620]
Please help. cannot render

![Screenshot (15)](https://user-images.githubusercontent.com/52619530/60768072-5a51b980-a0de-11e9-8d1a-c77c5bf314b3.png)

**Comment by PythonTryHard** [comment]
You can follow the guide [here](https://www.wikihow.com/Install-FFmpeg-on-Windows) on how to install FFmpeg. I've checked and it is in fact built from the [original repository](https://github.com/FFmpeg/FFmpeg).

**Comment by Elteoremadebeethoven** [comment]
I can't see the complete command prompt.

**Comment by girishgargcool** [comment]
Check Alexander

![Screenshot (16)](https://user-images.githubusercontent.com/52619530/60782163-3ab0a480-a163-11e9-9e0b-200b5aca2efb.png)

**Comment by Elteoremadebeethoven** [comment]
Share the COMPLETE command prompt, not only half.

**Comment by girishgargcool** [comment]
this is complete cmd

**Comment by Elteoremadebeethoven** [comment]
Sorry, you are right, my bad. Where did you locate ffmpeg?

**Comment by girishgargcool** [comment]
![Screenshot (17)](https://user-images.githubusercontent.com/52619530/60782520-a34c5100-a164-11e9-804c-fe0515d59c79.png)

**Comment by girishgargcool** [comment]
![Screenshot (18)](https://user-images.githubusercontent.com/52619530/60782579-e1497500-a164-11e9-80c5-f27aecd8e49c.png)

**Comment by Elteoremadebeethoven** [comment]
Ok, show a screenshot of the Environment Variables, I mean, all variables in the PATH file

**Comment by girishgargcool** [comment]
C:\Program Files (x86)\Intel\TXE Components\TCS\;C:\Program Files\Intel\TXE Components\TCS\;%SystemRoot%\system32;%SystemRoot%;%SystemRoot%\System32\Wbem;%SYSTEMROOT%\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Common Files\Roxio Shared\DLLShared\;C:\Program Files (x86)\Common Files\Roxio Shared\10.0\DLLShared\;C:\MinGW\bin;C:\Program Files\MiKTeX 2.9\miktex\bin\:C:\ffmpeg\bin:C:\sox\sox-14-4-2

**Comment by girishgargcool** [comment]
![Screenshot (20)](https://user-images.githubusercontent.com/52619530/60782694-5cab2680-a165-11e9-9f4a-e5d18993faa2.png)

**Comment by girishgargcool** [comment]
To check that I have installed ffmpeg

![Screenshot (19)](https://user-images.githubusercontent.com/52619530/60782716-6896e880-a165-11e9-933f-76028729933a.png)

**Comment by Elteoremadebeethoven** [comment]
I see, you use ":" symbols, you have to use ";".
You wrote:
`:C:\ffmpeg\bin:C:\sox\sox-14-4-2`
And it is:
`;C:\ffmpeg\bin;C:\sox\sox-14-4-2`

**Comment by girishgargcool** [comment]
okay, let me try again

**Comment by Elteoremadebeethoven** [comment]
If it works don't forget close the issue :D.

**Comment by girishgargcool** [comment]
awesome! its working. thanks a lot.

**Comment by girishgargcool** [comment]
I am on my way to master manim. I will disturb you again if I encounter any problem. I hope you don't mind it. LOTS OF LOVE FROM INDIA!!!

**Comment by Elteoremadebeethoven** [comment]
If you have any problem FIRST look by your self, if you don't find a solution then you can send a issue. You have to put a little of effort.
